src/blocks/stars.css

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars:last-of-type {
+  padding-bottom: 0;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../images/star.svg);
+  background-position: center;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
```

Comment:
Avoid using tag names for styling. Instead of using ':first-child' and ':nth-child', consider using class selectors to target specific elements. This will improve performance and maintainability.